Origin: upstream, https://github.com/cyrusimap/cyrus-sasl/commit/b764cd24835db090d8a375b0a539844523159145
From: Andreas Hasenack <panlinux@gmail.com>
Date: Mon, 10 May 2021 18:52:17 -0300
Subject: Partial fix for issue #624

This commit just transforms my comment in
https://github.com/cyrusimap/cyrus-sasl/issues/624#issuecomment-701534733
into an actual commit in a branch.

Signed-off-by: Andreas Hasenack <panlinux@gmail.com>
---
 docsrc/exts/sphinxlocal/builders/manpage.py | 1 -
 docsrc/exts/sphinxlocal/roles/saslman.py    | 1 -
 docsrc/exts/sphinxlocal/writers/manpage.py  | 9 +++------
 3 files changed, 3 insertions(+), 8 deletions(-)

diff --git a/docsrc/exts/sphinxlocal/builders/manpage.py b/docsrc/exts/sphinxlocal/builders/manpage.py
index a6281f79..126839e0 100644
--- a/docsrc/exts/sphinxlocal/builders/manpage.py
+++ b/docsrc/exts/sphinxlocal/builders/manpage.py
@@ -21,7 +21,6 @@
 from sphinx import addnodes
 from sphinx.errors import SphinxError
 from sphinx.builders import Builder
-from sphinx.environment import NoUri
 from sphinx.util.nodes import inline_all_toctrees
 from sphinx.util.console import bold, darkgreen
 from sphinx.writers.manpage import ManualPageWriter
diff --git a/docsrc/exts/sphinxlocal/roles/saslman.py b/docsrc/exts/sphinxlocal/roles/saslman.py
index f881d98f..bcafeece 100644
--- a/docsrc/exts/sphinxlocal/roles/saslman.py
+++ b/docsrc/exts/sphinxlocal/roles/saslman.py
@@ -18,7 +18,6 @@
 import re
 
 def setup(app):
-    app.info('Initializing saslman plugin')
     app.add_crossref_type('saslman', 'saslman', '%s', nodes.generated)
     return
 
diff --git a/docsrc/exts/sphinxlocal/writers/manpage.py b/docsrc/exts/sphinxlocal/writers/manpage.py
index 463524a4..3a939ec0 100644
--- a/docsrc/exts/sphinxlocal/writers/manpage.py
+++ b/docsrc/exts/sphinxlocal/writers/manpage.py
@@ -13,8 +13,9 @@
 """
 
 from docutils import nodes
+from time import strftime
+
 from sphinx.writers.manpage import (
-    MACRO_DEF,
     ManualPageWriter,
     ManualPageTranslator as BaseTranslator
 )
@@ -22,7 +23,6 @@
 
 from sphinx import addnodes
 from sphinx.locale import admonitionlabels, _
-from sphinx.util.osutil import ustrftime
 from sphinx.util.compat import docutils_version
 
 class CyrusManualPageWriter(ManualPageWriter):
@@ -68,15 +68,12 @@ def __init__(self, builder, *args, **kwds):
         if builder.config.today:
             self._docinfo['date'] = builder.config.today
         else:
-            self._docinfo['date'] = ustrftime(builder.config.today_fmt
+            self._docinfo['date'] = strftime(builder.config.today_fmt
                                               or _('%B %d, %Y'))
         self._docinfo['copyright'] = builder.config.copyright
         self._docinfo['version'] = builder.config.version
         self._docinfo['manual_group'] = builder.config.project
 
-        # since self.append_header() is never called, need to do this here
-        self.body.append(MACRO_DEF)
-
         # overwritten -- don't wrap literal_block with font calls
         self.defs['literal_block'] = ('.sp\n.nf\n', '\n.fi\n')
 
